PullRequestCreated/ReadMe.md

```diff
@@ -25,6 +25,16 @@ Token to access the Jira API.
 
 Jira project key that is used to create new issues.
 
+### `additional-fields`
+
+Additional fields to set when creating the Jira issue. 
+
+```yaml
+additional-fields: '{ "fixVersions": [ { "id": "15601" } ] }'
```

Comment:
FixVersion is a short live value that changes often, therefore it's not very suitable for the demo here.  Can you use something else, like label, component, or priority? Something that can be done in a simple way.